schema/cassandra/cadence/versioned/v0.29/processing_queue_states.cql

```diff
@@ -0,0 +1,14 @@
+CREATE TYPE domain_filter (
+    domain_ids    set<uuid>,
+    reverse_match boolean,
+    );
+
+CREATE TYPE processing_queue_state (
+    processing_queue_level  int,
+    ack_level               bigint,
+    max_level               bigint,
+    domain_filter           frozen<domain_filter>,
+);
+
+ALTER TYPE shard ADD cluster_transfer_processing_queue_state map<text, frozen<processing_queue_state>>;
```

Comment:
The might be more than one processing queue per shard per cluster, so I think the type should be a map of list? Another idea is that we can just store a blob here and perform encoding & decoding in persistence layer.

Reply:
Yup, will fix this.